test/proxy/transparent/ProxyAdmin.test.js

```diff
@@ -79,8 +58,8 @@ contract('ProxyAdmin', function (accounts) {
     context('with authorized account', function () {
       it('upgrades implementation', async function () {
         await this.proxyAdmin.upgrade(this.proxy.address, this.implementationV2.address, { from: proxyAdminOwner });
-        const implementationAddress = await this.proxyAdmin.getProxyImplementation(this.proxy.address);
-        expect(implementationAddress).to.be.equal(this.implementationV2.address);
+        const dummy = await new ImplV2(this.proxy.address);
+        expect(await dummy.version()).to.be.equals('V2');
```

Comment:
This is good, but we should also read from the EIP-1967 storage slot to check the implementation was set.